TeamForward/client/src/components/ImageIcon.js

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import blankProfileImg from "../../src/assets/home/blank-profile.png"
+
+const ImageIcon = (props) => {
+    const {user, imgClassName, ...otherProps} = props;
+    const [userPicture, setUserPicture] = useState("");
+
+    useEffect(()=>{
+        if(user.s3ProfilePhotoKey){
```

Comment:
if user is null this would cause a null pointer exception

Reply:
> if user is null this would cause a null pointer exception

based on reading the else on line 22 should avoid the error or is it the useEffect and it needs to be inside the if?
https://www.togidev.com/understanding-nullpointerexception-in-javascript-and-how-to-fix-it/